supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesTask.scala

```diff
@@ -392,7 +390,7 @@ final case class Resources(cpu_cores: Option[Int],
                            backend_parameters: Option[Map[String, Option[String]]]
 )
 
-final case class OutputFileLog(url: String, path: String, size_bytes: Int)
+final case class OutputFileLog(url: String, path: String, size_bytes: Option[Int])
```

Comment:
Simplification:

In the Funnel source code, `size_bytes` is [marked](https://github.com/ohsu-comp-bio/funnel/blob/master/tes/tes.pb.go#L1221) as `omitempty`, which [means](https://pkg.go.dev/encoding/json#Marshal),

> The "omitempty" option specifies that the field should be omitted from the encoding if the field has an empty value, defined as false, 0, a nil pointer, a nil interface value, and any empty array, slice, map, or string.

This validates my theory that the missing field represents `0` size. We can leave this as an `Int` and assume `0` if it's missing.
```suggestion
final case class OutputFileLog(url: String, path: String, size_bytes: Int)
```